e2e/ios/E2EOktaReactNative.xcodeproj/project.pbxproj

```diff
@@ -201,7 +201,6 @@
 			buildConfigurationList = 13B07F931A680F5B00A75B9A /* Build configuration list for PBXNativeTarget "E2EOktaReactNative" */;
```

Comment:
Changes in this file are from https://docs.expo.dev/bare/upgrade/. I chose from expo 49 to expo 50, and I got the changes needed for the xcode project file